### PR TITLE
Added DI usage in Schema

### PR DESCRIPTION
**Pull request by manchenkoff**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #9740

I've added DI usage instead of **new** keyword to get instances of QueryBuilder and ColumnSchemaBuilder, as was suggested in the discussion in #9740.

Now we can register different classes in container by
```php
\Yii::$container->set(
    \yii\db\sqlite\QueryBuilder::class, 
    ['class' => \santilin\sqlite\QueryBuilder::class]
);
```

### CODE REVIEW

File: framework/db/Schema.php [hunk 1]
```diff
@@ -313,7 +313,7 @@ public function refreshTableSchema($name)
      */
     public function createQueryBuilder()
     {
-        return new QueryBuilder($this->db);
+        return Yii::createObject(QueryBuilder::className(), [$this->db]);
```
Comment (bizley) [inline framework/db/Schema.php:316]: IIRC not using createObject was on purpose to make it faster. @samdark ?

Comment (bizley) [reply framework/db/Schema.php:316]: Maybe @rob006 remembers.

Comment (rob006) [reply framework/db/Schema.php:316]: I don't think that efficiency is a concern here. `QueryBuilder` is created only once per `Schema` instance. And AFAIK `ColumnSchemaBuilder` is used only in migration-like context, so it should be super rare anyway and any overhead in these scenarios should negligible.

**Comment by samdark** [comment]
👍

File: framework/db/sqlite/Schema.php [hunk 2]
```diff
@@ -16,7 +17,6 @@
 use yii\db\Expression;
 use yii\db\ForeignKeyConstraint;
 use yii\db\IndexConstraint;
-use yii\db\SqlToken;
```
Comment (rob006) [inline framework/db/sqlite/Schema.php:19]: This looks incorrect. `SqlToken` is used in typehint:
https://github.com/yiisoft/yii2/blob/2bdfac4ab7af3f68df494eab450ad4cd971ff7e5/framework/db/sqlite/Schema.php

Comment (samdark) [reply framework/db/sqlite/Schema.php:19]: https://github.com/yiisoft/yii2/commit/51fc314f585ee28dd9ce5a4fe5ed5ba9eefabb89

**Pull request merged**